kubernetes-client/src/test/java/io/fabric8/kubernetes/client/ConfigTest.java

```diff
@@ -234,7 +230,7 @@ public void testWithKubeConfig() {
     assertEquals("https://172.28.128.4:8443/", config.getMasterUrl());
     assertEquals("testns", config.getNamespace());
     assertEquals("token", config.getOauthToken());
-    assertTrue(config.getCaCertFile().endsWith("testns/ca.pem"));
```

Comment:
@mgorniew : How does this test the change you made in `CustomResourceOperationsImpl`?